tutorial/main.go

```diff
@@ -1,57 +1,44 @@
 package main
 
 import (
-	"bytes"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
-	"os/exec"
-	"strings"
 
 	"github.com/dadosjusbr/executor"
 )
 
 func main() {
 	goPath := os.Getenv("GOPATH")
-	repo := fmt.Sprintf("%s/src/github.com/dadosjusbr/coletores", goPath)
-
-	cmdList := strings.Split("git rev-list -1 HEAD", " ")
-	cmd := exec.Command(cmdList[0], cmdList[1:]...)
-	cmd.Dir = repo
-	var out bytes.Buffer
-	cmd.Stdout = &out
-	if err := cmd.Run(); err != nil {
-		log.Fatal(err)
-	}
-
-	collectBuildEnv := map[string]string{
-		"GIT_COMMIT": out.String(),
-	}
+	repo := fmt.Sprintf("%s/src/github.com/dadosjusbr/executor/tutorial", goPath)
```

Comment:
O que você tem aqui não é um repositório, mas um diretório local para a pasta raiz onde o tutorial está.

Pensei que o repo seria usado para fazer um clone do repositório. Se sim, tentar fazer clone isso deveria dar erro. Poderia explicar melhor, por favor?

Reply:
O _"repo"_ que é recebido aqui é o caminho completo até o repositório do codigo que se deseja executar. Se estivermos de acordo com esse conceito, talvez possamos até mudar o nome. Mas desde de o início tenho pensado nessa recepção como o caminho completo mesmo.

Até o momento, tinha pensado que a `Alba`, ao fazer o cadastro dos coletores, deveria fazer o clone e assim e manter a estrutura de arquivos daquilo que ela vai executar.

Imaginei o executor de forma mais isolada mesmo, apenas executando e retornando erros. Na vdd, imaginei alguns outros recursos para ele (como um configuração para repetir os estágios e enviar email), mas acho que entramos nesses detalhes em uma próxima reunião. 

Mas tentanto explicar o que você me perguntou. Seria isso mesmo, a `Alba` ou qualquer cliente do `Executor` deve garantir a existência dos arquivos(inclusive a info `GIT_COMMIT`, no caso dos coletores) e passar o caminho completo e demais configuração para o `Executor`.

Mas aguardp teu feedback sobre.

Reply:
Entendi a ideia. Tá beleza por mim. Vamos que vamos.

Depois me manda um PR mudando o campo para LocalBaseDir ou algo assim

Reply:
Certo. Pode deixar.